charts/datadog/ci/kubeconform-values.yaml

```diff
@@ -36,7 +36,7 @@ clusterAgent:
   enabled: true
   createPodDisruptionBudget: true
   nodeSelector:
-    kubernetes.io/os: linux
+    disktype: ssd
```

Comment:
I think the [CI is failing](https://github.com/DataDog/helm-charts/actions/runs/7872214797/job/21479319052?pr=1199#step:6:43) because we use this file in the CI Kind cluster. but the node doesn't have this nodeSelector. 

my suggestion for now is to rename this file: `charts/datadog/ci/kubeconform.yaml` so it will not be selected for the CI installation.

Reply:
I did that, but failed because I hadn't renamed it for each directory. But I do think there's some value to making sure to test / lint the values files, so since it looks like you've got a different / better fix in #1316 anyway, I reverted the change.

If you don't want it to be setting the disk type, I think we could probably set something arbitrary or harmless instead?

Reply:
Thanks @wyardley 

I'm still on it with another PR https://github.com/DataDog/helm-charts/pull/1317. Hopefully it will be the last one :)